File: NextGenCoder.java
import java.util.Scanner;

public class NextGenCoder {
    public static int CountCharacter(String str1,String str2){
        int count=0;
        for(int i=0,j=0;i<str1.length() && j<str2.length();i++,j++){
            if(Character.toLowerCase(str1.charAt(i))==Character.toLowerCase(str2.charAt(j))){
                count++;
            }
        }
        return count;
    }
    public static void main(String[] args) {
        String str1,str2;
        Scanner sc=new Scanner(System.in);
        str1=sc.next();
        str2=sc.next();
        System.out.println(CountCharacter(str1, str2));
        sc.close();
    }
}

File: NextGenCoder.py
import string
def CountCharacter(str1,str2):
    i=j=0
    n=len(str1)
    m=len(str2)
    count=0
    while i<n and j<m:
        if str1[i]==str2[j]:
            count+=1
        i+=1
        j+=1
    return count

str1=input()
str2=input()
print(CountCharacter(str1.lower(),str2.lower()))